src/ig/IGExporter.ts

```diff
@@ -1151,18 +1154,19 @@ export class IGExporter {
   addIgIni(igPath: string): void {
     const inputIniPath = path.join(this.igDataPath, 'ig.ini');
     if (this.config.template != null) {
+      const filePathString = path.join(path.basename(this.igDataPath), 'ig.ini');
       if (existsSync(inputIniPath)) {
-        const filePathString = path.join(path.basename(this.igDataPath), 'ig.ini');
-
```

Comment:
I can't figure out why these lines had to switch. What am I missing?

Reply:
Oh.  That's easy.  Because at one point in development (some commit in this multi-commit PR) I was using `filePathString` in the `else` clause.  Then I stopped -- and never rescoped `filePathString` back to the narrower scope.  Oops.  I can do that.